library/core/src/cell.rs

```diff
@@ -569,12 +569,17 @@ impl<T> Cell<[T]> {
     }
 }
 
+#[cfg(feature = "debug_refcell")]
+type CallerLoc = Cell<Option<&'static crate::panic::Location<'static>>>;
+
 /// A mutable memory location with dynamically checked borrow rules
 ///
 /// See the [module-level documentation](self) for more.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct RefCell<T: ?Sized> {
     borrow: Cell<BorrowFlag>,
+    #[cfg(feature = "debug_refcell")]
+    first_caller: CallerLoc,
```

Comment:
I'd just write out the type here, instead of using an alias that's only used once. (Saves a `#[cfg]` too.)